sawtooth_sdk/processor/core.py

```diff
@@ -75,6 +77,14 @@ def add_handler(self, handler):
         """
         self._handlers.append(handler)
 
+    def set_header_style(self, style):
+        """Sets a flag to indicate validator that transaction header field
```

Comment:
This comment is worded oddly. Should "validator" be removed?

Reply:
Ok

Reply:
This is addressed. I reworded the comment as the earlier didn't make sense.